cli/kata-env.go

```diff
@@ -181,6 +183,7 @@ func getRuntimeInfo(configFile string, config oci.RuntimeConfig) RuntimeInfo {
 		Config:              runtimeConfig,
 		Path:                runtimePath,
 		DisableNewNetNs:     config.DisableNewNetNs,
+		Experimental:        config.Experimental,
```

Comment:
Would be good to add the list of experimental features as well.

Reply:
I didn't get it. They've been added here, right?

Reply:
ah, ignore this. This was from before where I thought experimental was just a boolean.

Reply:
This information isn't being displayed by `kata-runtime kata-env`. It might also be useful to add it somewhere like:

- https://github.com/kata-containers/runtime/blob/master/cli/main.go#L330

Reply:
@jodh-intel It's being displayed only when "Experimental" isn't empty.

I'll add the log.